frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -467,10 +468,12 @@ const defaultFilters = {
   },
 };
 
+const none = 'None';
```

Comment:
Not quite what I meant :) For one, this will fail the linter as it's defined after its use... For two, this won't help using two different variables to match. You either need to use this exclusively... or use the _enum_ we talked about.

Reply:
Sorry for that. Now using `groupByTypes.None` everywhere instead of `'None'`